### PR TITLE
Adding support button - replacing share feedback button

### PR DESCRIPTION
**Pull request by manuthecoder**
**Summary**:
This PR addresses issue #7413 by replacing the existing feedback button with a new one that opens a menu. The aim is to streamline the footer by reducing clutter.

**Changes**:

- Replaces the standalone feedback button with a menu-triggering button to declutter the footer.
- Hides the Rails version information in the footer for a cleaner appearance.

**Preview**:
image
image

### CODE REVIEW

File: app/views/application/_footer.html.erb [hunk 1]
```diff
@@ -27,7 +15,7 @@
     <% unless Rails.env.production? %>
       <% @sql_count = QueryCount::Counter.counter %>
       <% @sql_cached_count = QueryCount::Counter.counter_cache %>
-      Loaded using <%= @sql_count %> unique SQL <%= "query".pluralize @sql_count %> (+ <%= @sql_cached_count %> cached). Running on Rails <%= Rails::VERSION::STRING %>.
```
Comment (sampoder) [inline app/views/application/_footer.html.erb]: We can keep this, it's in development only

File: app/views/application/_feedback_form.html.erb
Comment (sampoder) [inline app/views/application/_feedback_form.html.erb:1]: Let's rename this partial

**Comment by sampoder** [comment]
Thoughts on reducing the font weight? How does that look?

File: app/views/application/_footer.html.erb (hunk 1 above)
Comment (manu-gurudath) [reply app/views/application/_footer.html.erb]: haha didnt know this changed - I remember seeing this in prod a while back

**Comment by manu-gurudath** [comment]
image

**Review by sampoder — APPROVED** [review]
Looks good to me; thank you so much for the work on this!

**Comment by dwahbe** [comment]
thanks so much @manuthecoder ! this is great. would you want to write a changelog post for this?

**Comment by manu-gurudath** [comment]
Sure, I can do that right now

**Comment by sampoder** [comment]
let’s also highlight the help center in that post, not too many people know
about it yet

**Comment by manu-gurudath** [comment]
Just sent the changelog draft to both of you!

**Pull request merged**